### ISSUE TITLE
Toggle Shortcut Options

### ISSUE THREAD

**Issue by ryanbelisle** [preciousforever/sketch-layer-visibility#12]
Me, and probably many others, have huge, complicated Sketch documents. I'm wondering if you could add some flexibility to what gets toggled on and off, by using multiple shortcut commands?

With the new release - all Pages & Symbols are turned on/off. This can cause Sketch to come to a halt with complicated documents.

**My ideas:**
- Shortcut #1 - Toggles all Pages & Symbols layers on/off (current functionality)
- Shortcut #2 - Toggles selected Page layers on/off only (not symbols)
- Shortcut #3 - Toggles selected Page layers on/off & only Symbols used on that page (possible?)
- Shortcut #4 - Toggles Symbols Page layers on/off without having to go to the Symbols page

Basically, this gives you the flexibility to turn things on/off as you please for different situations.

I use your plugin daily, and am constantly turning toggling the layers on/off. Mainly, I use it to turn on/off what I call "spec blocks". These blocks show spacing between elements and assist developers.
